core/static/css/main.css

```
@@ -12,7 +12,7 @@ h1, h2, h3, h4, strong {
     color: var(--title-text-color);
 }
 
-.card, .modal-content {
+.card, .modal-content, .list-group-item {
     background-color: var(--body-alt-bg-color);
 }
 
```

Good fix! Perhaps we can also change the card header background colour so that it's slightly darker than the card body (just like in the light theme)
```suggestion
.card-header {
    background-color: var(--body-alt-dark-bg-color);
}
```